.github/dependabot.yml

```
@@ -13,11 +13,25 @@ updates:
   ignore:
   # Always ignore elasticsearch, future versions are always incompatible with our provider
   - dependency-name: "elasticsearch"
-  # These update basically every day, and 99.9% of the time we don't care
-  - dependency-name: "boto3"
-  - dependency-name: "boto3-stubs"
-  - dependency-name: "botocore"
-  - dependency-name: "botocore-stubs"
+  groups:
+    # Items in these groups are combined in a single PR
+    psycopg:
+      patterns:
+        # Combine both `psycopg` and `psycopg[c]`
+        - "psycopg*"
+- package-ecosystem: pip
+  directory: "/"
+  schedule:
+    interval: monthly
+  groups:
+    # Items in these groups are combined in a single PR
+    boto3:
+      patterns:
+        - "boto3*"
+    botocore:
+      patterns:
+        - "botocore*"
```

```suggestion
    boto-deps:
      patterns:
        - "boto3*"
        - "botocore*"
```
Would it make sense to tie both of these together as well?

---

Yeah, I'm fine with that too!